src/lib389/lib389/cli_conf/schema.py

```diff
@@ -383,3 +398,11 @@ def create_parser(subparsers):
     validate_parser.add_argument('DN', help="Base DN that contains entries to validate")
     validate_parser.add_argument('-f', '--filter', help='Filter for entries to validate.\n'
                                                         'If omitted, all entries with filter "(objectclass=*)" are validated')
+
+    import_oldap_schema_parser = schema_subcommands.add_parser('import-openldap-schema-file',
```

Comment:
Could we shorten the arg name?  "import-openldap"?  It's already under the schema subcommand, so saying "schema file" is a bit redundant.  This is not a hard request, just talking out loud :-)

Reply:
I give my ack regardless, up to you if you think shortening the arg is worth it or not.

Reply:
I thought about it. I think if I was an "admin" looking for this and I saw:

```
dsconf localhost schema import-openldap
VS
dsconf localhost schema import-openldap-schema-file
```

I'd "assume" that the first command may import *all* the openldap schema, where the second is just one file. So maybe a middle group is `schema import-openldap-file` ?

Reply:
It will be done! I'll merge once I make the change :)